### PR TITLE
this commit from feature-1 branch

### PR DESCRIPTION
**Pull request by SherefProgrammer**
i added some small changes in programm.cs file please review and confirm

### CODE REVIEW

**Review by shereffarag — COMMENTED** [review]
add your name to this message in code

File: helloworld/Program.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@
     // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
     app.UseHsts();
 }
-//aaaa
+//hello every one
```
Comment (shereffarag) [inline helloworld/Program.cs:15]: add your name here

File: helloworld/Program.cs [hunk 2]
```diff
@@ -12,7 +12,7 @@
     // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
     app.UseHsts();
 }
-//aaaa
+//hello every one sheref programmer
```
Comment (shereffarag) [inline helloworld/Program.cs:15]: greate

Comment (shereffarag) [inline helloworld/Program.cs:15]: good jos

File: helloworld/Program.cs [hunk 3]
```diff
@@ -12,7 +12,7 @@
     // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
     app.UseHsts();
 }
-//aaaa
```
Comment (shereffarag) [inline helloworld/Program.cs:15]: remove this one

Comment (SherefProgrammer) [reply helloworld/Program.cs:15]: already not in feature-1 branch this is in main branch

File: helloworld/Program.cs (hunk 2 above)
Comment (shereffarag) [inline helloworld/Program.cs:15]: add full name

Comment (shereffarag) [inline helloworld/Program.cs:15]: below add another commit

File: helloworld/Program.cs [hunk 4]
```diff
@@ -12,7 +12,8 @@
...
+//hello every one sheref programmer farag
```
Comment (SherefProgrammer) [inline helloworld/Program.cs:15]: add full name

File: helloworld/Program.cs [hunk 5]
```diff
@@ -12,7 +12,8 @@
...
+// adam sheref farag
```
Comment (SherefProgrammer) [inline helloworld/Program.cs:16]: add another name

**Review by shereffarag — APPROVED** [review]

**Comment by shereffarag** [comment]
merge feature-1 branch to main with edit in programm.cs file only

**Pull request merged**